<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomScan</class>
 <widget class="QDialog" name="CustomScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#550000;&quot;&gt;CyberGod KSGMPRH Custom Scan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="customScan_fileLocationLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="customScan_comboBox">
       <item>
        <property name="text">
         <string>file</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>folder</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="customScan_scanButton">
       <property name="text">
        <string>SCAN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="customScan_currentlyScanningLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="customScan_threatTable">
     <column>
      <property name="text">
       <string>Threat Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File Location</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
